autobahn/wamp_rawsocket_transport.hpp

```diff
@@ -0,0 +1,237 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Copyright (c) Tavendo GmbH
```

Comment:
Given the extent of your changes, I'm pretty sure we should add Topology copyright to all of those files that you're adding or rewriting (in addition to Tavendo copyright), e.g.:
`Copyright (c) 2015, 2016 Topology LP`

@oberstet, I suggest you add years for Tavendo as well, otherwise it gets hard to tell when the copyright expires. (This is making the entirely reasonable & unquestionable assumption that in 70 or 95 years or however long copyright will be then, autobahn-cpp will be in wide use as a foundational piece of all software.)